provider/cmd/crd2pulumi/gen/python.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2016-2020, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gen
+
+import (
+	"bytes"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/pulumi/pulumi/pkg/v2/codegen/python"
+	pschema "github.com/pulumi/pulumi/pkg/v2/codegen/schema"
+	"github.com/pulumi/pulumi/sdk/v2/go/common/util/contract"
+)
+
+func (pg *PackageGenerator) genPython(types map[string]pschema.ObjectTypeSpec, baseRefs []string) (map[string]*bytes.Buffer, error) {
```

Comment:
Note: In order to get the latest codegen, you'll need to update `go.mod` to refer to a more recent tag/commit. v2.9.2 was recently tagged, so that would work.

In go.mod:

```
	github.com/pulumi/pulumi/pkg/v2 v2.9.2
	github.com/pulumi/pulumi/sdk/v2 v2.9.2
```